.travis.yml

```diff
@@ -1,27 +1,25 @@
+sudo: required
```

Comment:
what? why? this is increasing build time by several minutes and starts full VM instead of just docker.